src/ngraph/op/experimental/dyn_broadcast.cpp

```diff
@@ -22,6 +24,13 @@ using namespace ngraph;
 
 const string op::DynBroadcast::type_name{"DynBroadcast"};
 
+ngraph::op::DynBroadcast::DynBroadcast(const Output<Node>& arg, const Output<Node>& shape)
+    : Op({arg, shape})
+{
+    add_default_broadcast_axes();
```

Comment:
This needs to be deferred until validation. Here set a flag that says no broadcast axes were provided and obey the flag in validate.